doc/classes/Array.xml

```diff
@@ -6,20 +6,35 @@
 	<description>
 		Generic array which can contain several elements of any type, accessible by a numerical index starting at 0. Negative indices can be used to count from the back, like in Python (-1 is the last element, -2 the second to last, etc.).
 		[b]Example:[/b]
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var array = ["One", 2, 3, "Four"]
 		print(array[0]) # One.
 		print(array[2]) # 3.
 		print(array[-1]) # Four.
 		array[2] = "Three"
 		print(array[-2]) # Three.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		var array = new Godot.Collections.Array{"One", 2, 3, "Four"}
```

Comment:
Interesting, I didn't know this syntax was possible. Anyway, you're missing a `;`

Reply:
fixed